tools/tm-signer-harness/README.md

```diff
@@ -1,5 +1,5 @@
 # tm-signer-harness
 
 See the [`tm-signer-harness`
-documentation](https://tendermint.com/docs/tools/remote-signer-validation.html)
+documentation](https://docs.tendermint.com/v0.37/tools/remote-signer-validation.html)
```

Comment:
Just a note here that `https://docs.tendermint.com/v0.37/` is not yet being built, so this will return a 404. But it's not a big deal in this specific instance because we should probably delete the entire tm-signer-harness tool anyways (this was deleted in v0.35, iirc).